data/signersDefault.json

```diff
@@ -85,7 +85,7 @@
 			"sdid": "aliexpress.com mail.aliexpress.com",
 			"ruletype": "ALL",
 			"priority": "DEFAULT_RULE_ALL_2"
-		}, {
+ 		}, {
```

Comment:
There seem to be a lot unintentionally whitespace changes.

Reply:
Sorry. I replaced the spaces with tabs (again)